### PR TITLE
feat: 🎸 add details page for api

### PR DESCRIPTION
**Pull request by flexbox**
✅ Closes: #68
✅ Closes: #62

### CODE REVIEW

**Comment by MatthysDev** [comment]
✅ Closes: https://github.com/flexbox/weshipit.today/issues/62

File: apps/weshipit/pages/react-native-tools/[slug].tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      seoDescription="The best tools and resources for busy developers"
+      ogImageTitle="React Native Serverless API"
+      withAccessoryRight={
+        <Button href="https://airtable.com/shrKPA2DGcG8xnQGG">
+          Add a new API
+        </Button>
+      }
+    >
+      <div className="m-auto w-2/3 pb-16">
+        <div className="m-auto my-8 w-2/3 text-2xl font-bold"> {name} </div>
```
Comment (flexbox) [inline apps/weshipit/pages/react-native-tools/[slug].tsx:30]: use `Text`

File: apps/weshipit/pages/react-native-tools/[slug].tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      </div>
+    </Layout>
+  );
+}
+
+export async function getServerSideProps({ params }) {
+  const apiKey = process.env.AIRTABLE_API_KEY;
+  const baseId = process.env.AIRTABLE_BASE_ID_REACT_NATIVE;
+
+  console.log('file: [slug].tsx:24 ~ getServerSideProps ~ params', params.slug);
```
Comment (flexbox) [inline apps/weshipit/pages/react-native-tools/[slug].tsx:50]: No logs 🙏

File: apps/weshipit/pages/react-native-tools/[slug].tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        ) {
+          records {
+            fields
+          }
+        }
+      }
+    `,
+  });
+
+  console.log('file: [slug].tsx:47 ~ getServerSideProps ~ data', data);
```
Comment (flexbox) [inline apps/weshipit/pages/react-native-tools/[slug].tsx:71]: No log 🙏

**Comment by flexbox** [comment]
@MatthysDev Missing recomanded tools 

image

**Comment by flexbox** [comment]
- [ ] responsive is broken 
image

- [ ] margin bottom is ugly 

image

**Pull request merged**